gradle.properties

```diff
@@ -30,16 +30,16 @@ rapis_version=22.12.0
 commons_cli_version=1.6.0
 commons_compress_version=1.26.1
 commons_csv_version=1.10.0
-commons_logging_version=1.3.0
+commons_logging_version=1.3.1
 gson_version=2.10.1
-jna_version=5.13.0
+jna_version=5.14.0
 slf4j_version=1.7.36
 log4j_slf4j_version=2.23.1
-awssdk_version=2.25.13
+awssdk_version=2.25.17
 hadoop_version=3.3.6
 javacpp_version=1.5.9
-javacv_version=1.5.9
-ffmpeg_version=6.0-1.5.9
+javacv_version=1.5.10
+ffmpeg_version=6.1.1-1.5.10
```

Comment:
They are using javacpp 1.5.10 as well, no upgrade for now.

Reply:
Done